scrapers/FragileSlave.yml

```diff
@@ -0,0 +1,46 @@
+name: "Fragile Slave"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fragileslave.com/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="videodetails"]/p[@class="date"]
+    scene:
+      Title:
+        selector: //div[@class="pagetitle"]//h1/text()
+      Details:
+        selector: //div[@class="videocontent"]/p/text()
+      Date:
+        selector: $details[1]/text()
+        postProcess:
+          - replace:
+              - regex: (\s*\|.*)
+                with:
+              - regex: (\d{2})\/(\d{2})\/(\d{4})
+                with: $3-$1-$2
+          - parseDate: 2020-02-01
+      Image:
+        selector: //div[@class='trailerblock'][1]/script
+        postProcess:
+          - replace:
+              - regex: .*<img.*src0_3x="(.*\.jpg)".*
+                with: $1
```

Comment:
In some scenes the `script` is not present `https://www.fragileslave.com/updates/trailers/Strapped-down-and-Climaxing.html` so you get no image
The images you get are too big imho, we should get the 1x or 2x by default
The regex needs to be more specific to work for both cases
The below seems to work ok for me
```yaml
      Image:
        # replace src0_1x in the below lines with src0_2x or src0_3x if you need higher res covers
        selector: //div[@class='trailerblock'][1]/script|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
        postProcess:
          - replace:
              - regex: .*<img.*src0_1x="([^"]+\.jpg)".*
                with: $1
```

Reply:
streamlined it even more and reduced the used image resolution